src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -476,9 +474,9 @@ private static IEnumerable<PathSegment> GetPathSegments(string httpRequestUri, I
             };
         }
 
-        public virtual Parameter BuildConstructorParameter(InputParameter operationParameter)
+        public Parameter BuildConstructorParameter(InputParameter operationParameter, TypeFactory typeFactory)
```

Comment:
considering this is an instance method, if we need typeFactory, could we just use the field `_typeFactory` directly? If so we do not need to change this signature.

Reply:
typeFactory is not in use anyway, removed.